src/routes/schedule/[slug]/+page.svelte

```diff
@@ -12,38 +12,44 @@
 </script>
 
 <LayoutWrapper>
-	<a class="bcu-button variant-filled-primary" href="/schedule">{`<-`} Back to Schedule</a>
-	<section class="flex flex-col gap-8">
-		<div class="pl-2 mb-4">
-			<span class="text-xl dark:text-primary-500">{`${fromDate} - ${toDate}`}</span>
-		</div>
-		<div class="bcu-card variant-ringed-primary p-8 flex flex-col gap-8">
-			<h1 class="font-bold">{schedule.title}</h1>
-			<div class="text-lg xl:text-xl opacity-80">{schedule.summary}</div>
+	<div class="flex m-2 md:m-8 justify-center">
+		<div class="flex flex-col gap-2 md:gap-12 max-w-6xl">
+			<a class="bcu-button variant-filled-primary" href="/schedule">{`<-`} Back to Schedule</a>
+			<section class="flex flex-col gap-8">
+				<div class="pl-2 mb-4">
+					<span class="text-xl dark:text-primary-500">{`${fromDate} - ${toDate}`}</span>
+				</div>
+				<div class="bcu-card variant-ringed-primary p-8 flex flex-col gap-8">
+					<h1 class="font-bold">{schedule.title}</h1>
+					<div class="text-lg xl:text-xl opacity-80">{schedule.summary}</div>
 
-			<h2 class="font-bold">Description</h2>
-			<div class="text-xl xl:text-2xl">
-				{schedule.body}
-			</div>
+					<h2 class="font-bold">Description</h2>
```

Comment:
I am not sure what is up here, do you have prettier turned on?

I think this looks good, but hard to tell with so many changes.

Reply:
Looks normal on my machine, so I'm wondering if that's just how github displayed the change? I've updated my repo and it looks like you can see those changes here. Let me know if you can't. I'm still figuring this stuff out.